Lib/socket.py

```diff
@@ -28,6 +28,8 @@
 socket.setdefaulttimeout() -- set the default timeout value
 create_connection() -- connects to an address, with an optional timeout and
                        optional source address.
+create_server() -- creates a TCP server socket and binds it to a specified
+                   address.
```

Comment:
I agree with Joannah. Also note that all lines in this section use the imperative mood.

```suggestion
create_server() -- create a TCP socket and bind it to a specified address.
```